src/widgets/forms.js

```diff
@@ -1208,32 +1243,35 @@ export function makeDescription (
     })
   }
 
-  const br = dom.createElement('br')
-  group.appendChild(br)
+  // const br = dom.createElement('br')
```

Comment:
**chore:** lets remove it if not needed anymore, commenting it out just leaves more clutter imho

Reply:
agree

Reply:
Yes.

Reply:
[timbl-forms 2ae2d0d] remove commented-out code lines